src/goto-instrument/nondet_static.cpp

```diff
@@ -46,6 +46,10 @@ void nondet_static(
       if(!ns.lookup(sym.get_identifier()).is_static_lifetime)
         continue;
 
+      // static that can be nondet initialized?
+      if(!ns.lookup(sym.get_identifier()).is_static_nondet_init)
```

Comment:
Could this replace the __CPROVER prefix check above? (These variables should all have static lifetime anyway?)